test/app-luatest/gh_7273_dt_new_err_on_cdata_test.lua

```diff
@@ -0,0 +1,24 @@
+local t = require('luatest')
+local g = t.group()
+
+local result_str = [[tarantool> require("datetime").new({timestamp = 1LL})
+---
+- error: 'builtin/datetime.lua:587: bad timestamp (''number'' expected, got ''cdata'')'
+...
+
+tarantool> ]]
+
+local TARANTOOL_PATH = arg[-1]
+
+g.test_dt_new_err_on_cdata = function()
+    local cmd = [[printf 'require("datetime").new({timestamp = 1LL})' | ]] .. 'INPUTRC=non_existent_file ' ..
+            TARANTOOL_PATH .. [[ -i 2>/dev/null]]
+
+    local fh = io.popen(cmd, 'r')
```

Comment:
It seems, there is no problem to test it without popen. Isn't it so?

Reply:
Reworked the test.